src/odemis/driver/smaract.py

```diff
@@ -2274,10 +2276,6 @@ def __init__(self, name, role, locator, ref_on_init=False, axes=None, speed=1e-3
         # Connect to the device
         self._id = c_uint32(0)
 
-        option_string = b"model " + str(SA_CTLDLL.hwModel) + "\n"
-        # options = c_char_p(option_string.encode("ascii"))
-        options = c_char_p("".encode("ascii"))
-
         logging.debug("Connecting to locator %s", locator)
         self.core.SA_CTL_Open(byref(self._id), c_char_p(locator.encode("ascii")), c_char_p(""))
```

Comment:
The second string also needs to be encoded:
`self.core.SA_CTL_Open(byref(self._id), c_char_p(locator.encode("ascii")), c_char_p("".encode('ascii'))`

Reply:
I concur.
I'd just remark that the second string can be more easily written `c_char_p(b"")`.